### DISCUSSION TITLE
IJCAI 2024 competition is using 【DrivAerNet】Dataset !

### DISCUSSION THREAD

**Discussion by wangguan1995** [Mohamedelrefaie/DrivAerNet#10] [Show and tell]
Hi there, thank you Mohamedelrefaie, Angela Dai, and Faez Ahmed for your contributions! You guys are doing a fantastic job~
We have used your datasets to host an open-source competition at IJCAI 2024. All competitors are striving to perform better using your dataset. Due to your excellent contributions, more and more users will refer to your repositories and papers!

- All competitors' code are posted and opened here : 

https://github.com/PaddlePaddle/PaddleScience/tree/develop/jointContribution/IJCAI_2024

- Some Pictures 

https://aistudio.baidu.com/projectdetail/7459168

![image](https://github.com/user-attachments/assets/15657c0e-a11b-47c6-88f4-3eebd8a6cbb8)

![image](https://github.com/user-attachments/assets/0cac65bc-b0a2-486d-a521-95174c575ae0)
<img width="618" alt="image" src="https://github.com/user-attachments/assets/47236652-778c-4677-a442-59324494505d">

![image](https://github.com/user-attachments/assets/f68427ca-b7f3-426c-b113-7059e2542451)

https://x.com/IJCAIconf/status/1822235037695156551

**Comment by Mohamedelrefaie** [comment]
Thank you so much for the kind words! We’re really excited to hear that DrivAerNet was used in the IJCAI 2024. It’s amazing to see the creative ways people are using the dataset, and we’re eager to see the results of everyone’s hard work.

We’re especially looking forward to how DrivAerNet++ might be used in future research and competitions. 

Thanks again for sharing this, and for all your efforts in making the competition a success!
